web/sites/default/config/core.entity_view_display.node.service_location.default.yml

```diff
@@ -40,14 +45,33 @@ mode: default
 content:
   field_body_content:
     type: text_default
-    weight: 4
+    weight: 6
     region: content
     label: hidden
     settings: {  }
     third_party_settings: {  }
+  field_emergency_closure:
+    weight: 1
+    label: hidden
+    settings:
+      format: default
+      format_custom_false: ''
+      format_custom_true: ''
+    third_party_settings: {  }
+    type: boolean
+    region: content
+  field_floor:
+    type: instagram_embed
```

Comment:
Is the floor field supposed to be "String"?

Reply:
Good catch! I'll get that fixed.

Reply:
Fixed.